core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -399,6 +404,16 @@ class MapUnit {
         tile.improvementInProgress = null
     }
 
+    private fun tryProvideProductionToClosestCity()
+    {
+        val tile = getTile()
+        val closestCity = civInfo.cities.minBy { it.location.dst(tile.position) }
+        if (closestCity == null) return
+        val productionToAdd = (25 - closestCity.location.dst(tile.position) * 4).toInt()
```

Comment:
Where does this come from? Couldn't find any sources about this.
But if is is dependent on distance to city, we should use closestCity.getCenterTile().aerialDistanceTo(tile)

Reply:
Yes, dependent on distance.
jungles yield 20 in this mod https://steamcommunity.com/sharedfiles/filedetails/?id=103271218 - same as forest author stated in description

https://steamcommunity.com/app/8930/discussions/0/46476145384069876/
>2 tiles away is still 20 and on third one is 17
https://forums.civfanatics.com/threads/how-far-from-city-can-chop-forests-for-production.487398/
> Haven't tested it, but I think I've chopped forests more than 5 tiles away (but maybe not). I've definitely chopped outside my culture borders and that gives fewer hammers for the same distance from the city center.

Current guess is` if (distance < 2) 20 else (20 - (distance - 2) * 3).toInt()`
probably some modifier for not being in your borders. Maybe it starts to decrease outside

Reply:
I asked for screenshots, I will science out exact formula

Reply:
![tooltip](https://user-images.githubusercontent.com/69697985/91944642-14ef4300-ed07-11ea-9fb8-9d8b2b20d2b3.png)
It shows amount in the tooltip. Era and production in city does not matter (in civ 5 vanilla).
If tile is owned by worker's civ - 20 if next to city, 20 if one tile away, 15 if two tiles away, then 10, 5, 0
2/3 of that if tile is not owned - 13, 13, 10, 6, 3, 0

Notification looks like this:
![NOTIFICATION](https://user-images.githubusercontent.com/69697985/91944629-0e60cb80-ed07-11ea-8250-e7cadf335344.jpg)